src/util.js

```diff
@@ -197,3 +191,58 @@ export const importantify = (string) =>
     string.replace(
         IMPORTANT_RE,
         (_, base) => base + " !important;");
+
+const getBrowserProperties = () => {
+    if (!getBrowserProperties.availableStyles) {
+        getBrowserProperties.availableStyles = {};
+        const styles = Object.keys(window.getComputedStyle(document.documentElement, ''));
+        for (let i = 0; i < styles.length; i++) {
+            const style = styles[i];
+            if (isNaN(Number(style))) {
+                getBrowserProperties.availableStyles[style] = style;
+            }
+        }
+    }
+    return getBrowserProperties.availableStyles;
+};
+
+const stylePrefixes = ['Moz', 'webkit', 'ms', 'O'];
+const getVendorPrefix = (property) => {
+    const validProperties = getBrowserProperties();
+
+    const validatedProp = validProperties[property];
+    if (validatedProp) {
+        return validatedProp;
+    }
+    const capitalProp = property[0].toUpperCase() + property.substr(1);
+    for (let i = 0; i < stylePrefixes.length; i++) {
+        const prefix = stylePrefixes[i];
+        const prefixedProperty = `${prefix}${capitalProp}`;
+        if (validProperties[prefixedProperty]) {
+      // learn which styles the browser likes
+            validProperties[validatedProp] = prefixedProperty;
```

Comment:
Let's assume you have a rule like `boxReflect`.
It first checks to see if the browser can handle that by looking up `boxReflect` (Line 213).
If that doesn't work, then it tries all the prefixes. `MozBoxReflect, webkitBoxReflect`, ah, found it!
We know that the browser doesn't support `boxReflect` but it does support `webkitBoxReflect`.
So now we add that to the validatedProperties table: `{boxReflect: 'webkitBoxReflect'}`.
So the next time you look up `boxReflect` (Line 213) you grab the prefixed version immediately without having to set up the for loop.  
The time savings is minimal, but aphrodite's target is IE10, so we can't use a `Set`, so we might as well make use of the object.